.github/workflows/update-test.yaml

```diff
@@ -35,6 +35,8 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Update tests ${{ matrix.pg }}
+      # skip update test for PG15 and above
+      if: matrix.pg < '15'
```

Comment:
Is the test disabled because there are no container images for PG 15 yet or are there other reasons?

Reply:
Yes, we skip because there is no container images for PG 15.